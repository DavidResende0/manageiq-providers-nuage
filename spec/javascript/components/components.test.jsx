import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { Spinner } from 'patternfly-react';

describe('<Spinner/>', () => {
  it('renders <Spinner> component', () => {
    expect(toJson(shallow(<Spinner loading />))).toMatchSnapshot();
  });
});
