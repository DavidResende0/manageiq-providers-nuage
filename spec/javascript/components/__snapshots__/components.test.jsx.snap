// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> renders <Spinner> component 1`] = `
<div
  className="spinner spinner-md"
/>
`;
